setup.py

```diff
@@ -1,4 +1,16 @@
 from setuptools import find_packages, setup
+import pkgutil
+
+install_requires = [
+    'Django>=1.8',
+    'django_otp>=0.3.4,<0.99',
+    'qrcode>=4.0.0,<4.99',
+    'django-phonenumber-field>=1.1.0,<1.99.0',
```

Comment:
Can you change this to `...<1.99` instead; so to be consistent with the other version notations?